tests/conftest.py

```diff
@@ -25,7 +25,15 @@
 def example_wflow_model():
     logger = logging.getLogger(__name__)
     root = join(EXAMPLEDIR, "wflow_piave_subbasin")
-    mod = WflowModel(root=root, mode="r", data_libs="artifact_data", logger=logger)
+    mod = WflowModel(
+        root=root,
+        mode="r",
+        data_libs=[
+            "artifact_data",
+            "https://github.com/Deltares/hydromt_wflow/releases/download/v0.5.0/wflow_artifacts.yml",
```

Comment:
For all three tests, why do we need this? I don't think you test the GLCNMO data so far no?

Reply:
That was indeed incorrectly copied from the demand branch. Removed it for this PR!